api/src/main/kotlin/edu/wgu/osmt/RoutePaths.kt

```diff
@@ -1,36 +1,51 @@
 package edu.wgu.osmt
 
+import org.apache.commons.lang3.StringUtils
+
 object RoutePaths {
+
+
+    const val LEGACY = "/v2"
+    const val LATEST = "/v3"
+    const val DEFAULT = LATEST
```

Comment:
Seems default is unused

Reply:
yeah, this constant was meant to be used for changing the default redirect, but we are not using that logic anymore, @Corpratespaz should we remove it?

Reply:
Yes, please remove this unused constant